crypto/build.info

```diff
@@ -5,6 +5,7 @@ SUBDIRS=objects buffer bio stack lhash rand evp asn1 pem x509 x509v3 conf \
         md2 md4 md5 sha mdc2 gmac hmac ripemd whrlpool poly1305 blake2 \
         siphash sm3 des aes rc2 rc4 rc5 idea aria bf cast camellia \
         seed sm4 chacha modes bn ec rsa dsa dh sm2 dso engine \
+        crmf \
```

Comment:
merge this with the following one.

Reply:
Done. I had put 'crmf' in an extra line such to stress that this is an addition and to avoid touching an existing line.